src/components/Aboutpage/styles.module.css

```diff
@@ -63,58 +67,55 @@ body{
 }
 
 
-.ap_cont_4_founder{
-    background: #1B2731;
-}
-
-.ap_cont_4_founder{
+.ap_cont_4_founder {
     display: flex;
     flex-direction: row;
+    background: #1B2731;
+    margin-bottom: 74px;
```

Comment:
DO you think making this relative would be better?

Reply:
I didn't get you 
Could you elaborate what do you mean by relative
Position relative or anything else

Reply:
position relative maybe in `%`

Reply:
You can coauthor this and I will commit your suggestion

Reply:
@bishal7679 this issue is being perfectly handled in the #175 PR by working on the footer component, If you want I can add you there as a collaborator